opentech/apply/projects/templates/application_projects/project_admin_detail.html

```diff
@@ -0,0 +1,60 @@
+{% extends "application_projects/project_detail.html" %}
```

Comment:
I just noticed the name, `application_projects` not sure that the `application` part is truly necessary.

Reply:
The CBVs use this app name when they build their `template_name` value.  We could override it but I'm not sure what it would gain us?